libs/cms/src/lib/models/powerBiSlice.model.ts

```diff
@@ -22,6 +22,9 @@ export class PowerBiSlice {
 
   @Field({ nullable: true })
   owner?: 'Fiskistofa'
+
+  @Field(() => graphqlTypeJson, { nullable: true })
+  powerBiEmbedPropsFromServer?: Record<string, unknown> | null
```

Comment:
It's actually not neccessary to define this here, if you don't need it (then you can skip it also in mapPowerBiSlice). The field resolver decorators adds it to the final graphql schema.